### ISSUE TITLE
LinkedInShareButton broken due to new API

### ISSUE THREAD

**Issue by ericding98** [nygardk/react-share#362]
I think LinkedIn's new share API dropped support of the old URL API and breaks the implementation here, can anyone confirm?

https://docs.microsoft.com/en-us/linkedin/consumer/integrations/self-serve/share-on-linkedin?context=linkedin/consumer/context

**Comment by mraghuram** [comment]
I have exactly the same issue. LinkedInShareButton needs to be updated. It's expecting https://www.linkedin.com/sharing/share-offsite/?url={URL}

I have just replaced LinkedShareButton with <a href=https://www.linkedin.com/sharing/share-offsite/?url={URL}>

**Comment by shrinathaithal** [comment]
> I have exactly the same issue. LinkedInShareButton needs to be updated. It's expecting [https://www.linkedin.com/sharing/share-offsite/?url={URL}](https://www.linkedin.com/sharing/share-offsite/?url=%7BURL%7D)
> 
> I have just replaced LinkedShareButton with <a href=[https://www.linkedin.com/sharing/share-offsite/?url={URL}>](https://www.linkedin.com/sharing/share-offsite/?url=%7BURL%7D%3E)

Raise a PR?

**Comment by hielfx** [comment]
@mraghuram your solution is not working for me... any idea why?

**Comment by bhatsudo** [comment]
+1 
Not working for me too.

**Comment by bhatsudo** [comment]
@hielfx check if the complete URL is there in your URL variable. It should start with https://....

**Comment by BrianDev1** [comment]
This does not seem to work on mobile browsers, anyone found a work around?

**Comment by Sergei-Karpuk98** [comment]
+

**Comment by 3DX900** [comment]
So ? is this button really broken ?

**Comment by theenoahmason** [comment]
Linked in share button is working for me - seems to be using opengraph meta tags on the target page to create the card.

I cannot, however, add predefined text. I have tried both "title" and "summary" props. It seems its expecting a "text" param.

**Comment by 3DX900** [comment]
Yes, it works for me. I just want to add a clarification that may help you, if you are on localhost, social networks do not accept the url, it may also be related to OpenGraph beacons.

**Comment by 3DX900** [comment]
It can be closed.

**Comment by carlos-soto** [comment]
> Linked in share button is working for me - seems to be using opengraph meta tags on the target page to create the card.
> 
> I cannot, however, add predefined text. I have tried both "title" and "summary" props. It seems its expecting a "text" param.

I am experiencing the same issue, can't seem to be able to send any text along with the URL.

**Comment by GoudekettingRM** [comment]
Ran into the same issue, the linkedin api is expecting a text param, not a summary or title. 

I made a custom link like so:

```tsx
      <Link
        href={`https://www.linkedin.com/sharing/share-offsite/?url=${url}&text=${message}`}
        onClick={(event) => {
          event.preventDefault(); // Prevent it behaving like a link
          window.open(event.currentTarget.href, 'linkedin-share', 'width=550,height=450'); // Trigger linked in to open in a new window
        }}
      >
        <LinkedinIcon size={32} round /> {/* Use the icon from react-share to have it look the same as the other share icons */}
      </Link>
```

**Comment by JasonLovesDoggo** [comment]
> Ran into the same issue, the linkedin api is expecting a text param, not a summary or title.
> 
> I made a custom link like so:
> 
>       <Link
>         href={`https://www.linkedin.com/sharing/share-offsite/?url=${url}&text=${message}`}
>         onClick={(event) => {
>           event.preventDefault(); // Prevent it behaving like a link
>           window.open(event.currentTarget.href, 'linkedin-share', 'width=550,height=450'); // Trigger linked in to open in a new window
>         }}
>       >
>         <LinkedinIcon size={32} round /> {/* Use the icon from react-share to have it look the same as the other share icons */}
>       </Link>

This is now broken too
